### DISCUSSION TITLE
Split Chat Covers UI

### DISCUSSION THREAD

**Discussion by Anwynn** [pieterjanbuntinx/kitten-tracker#23] [General]
The UI is covered by private chat when split chat is enabled.

**Comment by pieterjanbuntinx** [comment]
Can you post a screenshot of this? I don't really know what you mean by split chat

**Reply by UncleWilly** [reply]
![image](https://user-images.githubusercontent.com/98077031/150281719-c936bdc4-d2d4-4469-b130-e47db3d95a55.png)

Would be great if the kitten tracker box could be moveable.
